General-Installing-Instructions.md

```diff
@@ -58,16 +58,43 @@ The following modules are optional, but preferred to be installed.
 - gmp (for plugin support)
 - com or dotnet (windows only)
 
-## Ports for FreeBSD
+## FreeBSD
 
-When installing on FreeBSD and variants, you must consider these packages.
+When installing on FreeBSD you can use two ways. Both ways have few pros and cons:
+- compiled packages - fast, but invariant dependencies (like older MySQL server, PHP version, ...)
 
-- `www/apache2`
-- `net/rrdtool`
-- `net/net-snmp`
-- `www/php-cgi`
-- `lang/php` (With MySQL and SNMP Support)
-- `databases/mysql-server`
+- FreeBSD ports - compilation could last long time, but without invariant dependencies
+
+For both ways, cacti has a lot of dependent packages, you don't need to install anything else. Everything is prepared.
+
+- For packages you need only:
+```sh
```

Comment:
This code block should all be indented by three characters to aline with the bullet point.